cosmpy/aerial/contract/__init__.py

```diff
@@ -51,14 +51,19 @@ def _generate_label(digest: bytes) -> str:
 
 class LedgerContract:
     def __init__(
-        self, path: str, client: LedgerClient, address: Optional[Address] = None
+        self,
+        client: LedgerClient,
+        path: Optional[str] = None,
+        address: Optional[Address] = None,
```

Comment:
```suggestion
        self,
        path: Optional[str],
        client: LedgerClient,
        address: Optional[Address] = None,
        digest: Optional[bytes] = None,
```

:point_up: lets keep the original signature for the function I think. Since we do really want to have the contract path so that we don't end up creating lots of duplicate contracts on the chain.

I have also added a `digest` parameter. This would be nice to include in the constructor here so that the use can specify a digest if they know it.